Libplanet/Libplanet.csproj

```diff
@@ -62,15 +62,6 @@ https://docs.libplanet.io/</Description>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Bencodex" Version="0.11.0" />
-    <PackageReference Include="Bencodex.Json" Version="0.11.0" />
-    <PackageReference Include="BouncyCastle.Cryptography" Version="2.0.0" />
-    <PackageReference Include="Caching.dll" Version="1.4.0.1" />
-    <PackageReference Include="Destructurama.Attributed" Version="2.0.0" />
-    <PackageReference Include="ImmutableTrie" Version="1.0.0-alpha" />
-    <PackageReference Include="LiteDB" Version="4.1.4" />
-    <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
-    <PackageReference Include="Planetarium.LruCacheNet" Version="1.2.0" />
```

Comment:
Not sure entirely removing dependency is great. This relies on knowing each subproject's dependency. 🤔

Reply:
Not sure for this, too. hmm...

Reply:
My preference is 50% and 50%.

Reply:
Well, I haven't thought about it too deeply, but if the project is well organized either
- Having different versions shouldn't matter
- Shouldn't reference the "lower" packages in the first place
If we are having trouble, I'd say that's generally indicative of some code smell.